boxwise_flask/models.py

```diff
@@ -1,18 +1,53 @@
 """Model definitions for database"""
-from peewee import CharField, CompositeKey, DateField, DateTimeField
+from peewee import CharField, CompositeKey, DateField, DateTimeField, IntegerField
 from playhouse.shortcuts import model_to_dict
 
 from .app import db
 
 
-class Person(db.Model):
-    firstname = CharField()
-    lastname = CharField()
-    camp_id = CharField()
-    id = CharField()
+class Stock(db.Model):
+    # INSERT INTO stock (
+    #   box_id, product_id, size_id, items, location_id,
+    #   comments, qr_id, created, created_by, box_state_id)
+    # VALUES (
+    #   :box_id, :product_id, :size_id, :items, :location_id, :comments,
+    #   :qr_id, :created, :created_by, :box_state_id)
+
+    id = IntegerField()
```

Comment:
I found something recently which suggested the primary key for a model doesn't need to be defined. If you delete this entry you still have access to the id implicitly.